llvm/test/CodeGen/AIE/aie2/schedule/postpipeliner/interleave-prologue.mir

```diff
@@ -0,0 +1,422 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
Clever! I like TDD. It also helps to see from where you start and how the work progresses with the sequence of commits.